notebooks/236-stable-diffusion-v2/236-stable-diffusion-v2-text-to-image.ipynb

```diff
@@ -2,10 +2,8 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #67.        # Quantize activations for operations without weights
</pre><p>The comment should be rephrased probably to <code># Quantize both weights and activations of Convolution lyaers.</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1709/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
Changed